go/client/cmd_simplefs_stat.go

```diff
@@ -56,6 +57,17 @@ func NewCmdSimpleFSStat(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.
 	}
 }
 
+func prefetchStatusString(e keybase1.Dirent) string {
+	if e.PrefetchStatus != keybase1.PrefetchStatus_IN_PROGRESS ||
```

Comment:
I think maybe it's confusing to say "sync in progress" only when there are 0/n bytes synced. Not sure if I have any better alternatives. "Sync waiting to start" or something?

Reply:
This is actually when 0 is the denominator, that is we haven't even fetched a single block yet to look at it.  I'm changing this to `keybase1.PrefetchStatus_NOT_STARTED.String()` in that case.